tfx/components/transform/executor_test.py

```diff
@@ -206,49 +207,33 @@ def testCounters(self):
   def testDoWithCache(self):
 
     # First run that creates cache.
-    output_cache_artifact = _OutputCache()
```

Comment:
so the cache is already supported in do(), just it is not exposed to component level?

Reply:
based on my current understanding yep. `cache_input_path` key was never provided in input_dict nor `cache_output_path` in output_dict since neither existed in the Transform component spec (thus why caching was not enabled yet)